### PR TITLE
MWPW-162758-Add cookie check for upload events

### PR DESCRIPTION
**Pull request by Ruchika4**
- Add cookie check for upload events before redirect

Resolves: [MWPW-162758](https://jira.corp.adobe.com/browse/MWPW-162758)

Test URLs:

Before: https://stage--cc--adobecom.hlx.page/acrobat/online/sign-pdf?unitylibs=stage
After: https://stage--cc--adobecom.hlx.page/acrobat/online/sign-pdf?unitylibs=analytics-cookie

Note: As the cookie is set only for adobe.com domain test by accessing above url on stage.adobe.com and overriding the action-binder.js with the code form this feature branch

### CODE REVIEW

**Comment by Ruchika4** [comment]
@TsayAdobe @joaquinrivero , could you please check once the wait time we have added and confirm if it should be ok. We are waiting for 2000 ms for cookie. We are also waiting for 500 ms before redirecting to product page when cookie not found so that the message can be logged to lana.

File: unitylibs/core/workflow/workflow-acrobat/action-binder.js [hunk 1]
```diff
@@ -251,11 +272,21 @@ export default class ActionBinder {
...
+        this.updateProgressBar(this.splashScreenEl, 100);
+        if (this.checkCookie()) {
+          window.location.href = response.url;
+        }
+        else {
+          await this.dispatchErrorToast('verb_cookie_not_set', 200, "Not all cookies found, redirecting anyway", true);
+          const sleep = (delay) => new Promise((resolve) => setTimeout(resolve, delay));
+          await sleep(500);
+          window.location.href = response.url;
+        }
```
Comment (TsayAdobe) [inline unitylibs/core/workflow/workflow-acrobat/action-binder.js:289]: It (L281-L289) looks like equivalent to the code below
```JS
        if (!this.checkCookie()) {
          await this.dispatchErrorToast('verb_cookie_not_set', 200, "Not all cookies found, redirecting anyway", true);
          await new Promise(r => setTimeout(r, 500));
        }
        window.location.href = response.url;
```

**Review by TsayAdobe — APPROVED** [review]

File: unitylibs/core/workflow/workflow-acrobat/action-binder.js (hunk 1 above)
Comment (Ruchika4) [reply unitylibs/core/workflow/workflow-acrobat/action-binder.js:284]: Thanks @TsayAdobe I have incorporated suggested change.

**Review by sanjayms01 — APPROVED** [review]

**Comment by Ruchika4** [comment]
Merging to this stage as this PR cannot be tested on hlx.page url without browser override.

cc @sanjayms01

**Pull request merged**